.travis.yml

```diff
@@ -7,10 +7,16 @@ go:
   - 1.10.x
   - 1.11.x
 
+env:
+  - GO111MODULE=on
+
 install:
-  - env GO111MODULE=on go get -v ./...
-  - env GO111MODULE=on go build ./...
+  - go get -v ./...
+  - go build ./...
   - go get github.com/onsi/ginkgo
   - go install github.com/onsi/ginkgo/ginkgo
 
-script: env GO111MODULE=on $HOME/gopath/bin/ginkgo -p -r --randomizeAllSpecs --failOnPending --randomizeSuites --race && env GO111MODULE=on go vet
+script: |
+  $HOME/gopath/bin/ginkgo -p -r --randomizeAllSpecs --failOnPending --randomizeSuites --race &&
+  go vet &&
+  [ -z "`gofmt -l -e -s -w .`" ]
```

Comment:
-l -> list changed files
-e -> all errors
-s -> simplify
-w -> rewrite (so users can run it locally and don't have to bother with the diff)
. -> recursive